packages/mdc-checkbox/README.md

```diff
@@ -255,8 +255,28 @@ not return an object.
 
 ## Theming
 
-MDC Checkboxes use the theme's primary color by default for checked and indeterminate states, and are completely dark theme
-aware.
+MDC Checkboxes use the theme's secondary color by default for "marked" states (i.e., checked or indeterminate), and are completely dark theme aware.
+
+### Sass Mixins
+
+The following mixins apply only to _enabled_ checkboxes. It is not currently possible to customize the color of a _disabled_ checkbox.
+
+Mixin | Description
+--- | ---
+`mdc-checkbox-stroke-and-container-fill-color($unmarked-stroke-color, $unmarked-container-fill-color, $marked-stroke-color, $marked-container-fill-color, $animate)` | Generates CSS classes to set and animate the stroke color and/or container fill color of a checkbox
```

Comment:
I hate using "and" in API names...

How about `mdc-checkbox-marked-indicator`?

Reply:
But it takes both marked and unmarked styles? Coming up with a good descriptive name for this is hard...

Reply:
Agreed, I'm not wild about the name either...

The mixin covers both marked and unmarked states, so putting `marked` in the name would probably be misleading. And I'm not sure I would call this an `indicator`, exactly.

Since _stroke_ and _fill_ are both properties of the container, what about:
`mdc-checkbox-container-colors`?

Making the mixin name plural will hopefully make it obvious that it takes multiple color arguments, and help differentiate it from the other theme mixins that only take one color argument.

Reply:
mdc-checkbox-container-colors sounds right to me.